src/commands/tutor-commands/tutor-leaderboard.ts

```diff
@@ -0,0 +1,54 @@
+import { SlashCommandBuilder } from "discord.js";
+import { Command } from "../../interfaces/Command";
+import { createEmbed } from "../../utils/embeded";
+import { getTutorLeaderboard } from "../../utils/supabase";
+import { commandLog, sendError } from "../../utils/logs";
+
+const tutorleaderboardBody = (array: string[]): string => {
+  let body = "";
+  array.forEach((slot) => (body = `${body}${slot}\n`));
+  return body;
```

Comment:
**suggestion:** refactor
**hint:** use `join()`
**comment:** KISS

Reply:
updated and pushed